packages/desktop/views/dashboard/collectibles/components/Erc721CollectibleDetails.svelte

```diff
@@ -43,6 +43,11 @@
             key: localize('general.tokenId'),
             value: tokenId,
         },
+        {
+            key: localize('general.url'),
+            value: nft.mediaUrl || undefined,
```

Comment:
Extract the mediaUrl from the `nft` object to have it consistent with the other variables.